src/user/mod.rs

```diff
@@ -277,7 +277,7 @@ round_trip! { raw::RawTwitterUser,
         /// "Perspectival" items within this tweet that depend on the authenticating user
         /// [may not be completely reliable][stale-embed] in this embed.
         ///
-        /// [stale-embed]: https://dev.twitter.com/docs/faq/basics/why-are-embedded-objects-stale-or-inaccurate
+        /// [stale-embed]: https://web.archive.org/web/20160617182407/https://dev.twitter.com/faq#41
```

Comment:
I could not find any documentation on this at developer.twitter.com, so resorted to archive.org.

Reply:
Hmm, this is strange. It looks like the `status` field isn't even mentioned in the documentation any more. I've asked [on the Twitter Community forum](https://twittercommunity.com/t/status-field-on-users/140024) about this. It may be prudent to just remove this field in the future, if they're not even going to acknowledge it.